doc/classes/TabContainer.xml

```diff
@@ -36,20 +35,21 @@
 			<return type="Popup">
 			</return>
 			<description>
+				Returns the [class Popup] node instance if one has been set already with [method set_popup].
 			</description>
 		</method>
 		<method name="get_previous_tab" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
-				Returns the previous tab index that was being shown.
+				Returns the previous tab index. Supports cycling indexes.
```

Comment:
index -> indices ?

Reply:
Linguistically speaking, they are equally valid phrasings in English, but I'm not sure if there is a convention we are using for the docs...

Reply:
`grep` returns 4 'indexes' and 14 'indices' in the current docs.

Reply:
Alrighty then, guess we'll go with 'indices'.

Reply:
done